pandas/tests/groupby/test_apply.py

```diff
@@ -866,3 +866,24 @@ def fct(group):
         [[1.0, 2.0], [3.0], [np.nan]], index=pd.Index(["a", "b", "none"], name="A")
     )
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("contains_na", [True, False])
+def test_apply_mean_with_nullable_integer(contains_na):
+    # https://github.com/pandas-dev/pandas/issues/32219
+    if contains_na:
```

Comment:
IMO you can put "values" as the values of the parameteration, with a comment explaining that one of the arrays is with nullable integers, and one is not.
then you can remove the ```if/else``` clause completely.

Reply:
Good idea